modules/charon-core/src/main/java/org/wso2/charon3/core/encoder/JSONDecoder.java

```diff
@@ -842,7 +842,7 @@ public SearchRequest decodeSearchRequestBody(String scimResourceString,
             return searchRequest;
 
         } catch (JSONException | IOException e) {
-            logger.error("Error while decoding the resource string");
+            logger.debug("Error while decoding the resource string");
```

Comment:
let's add e to this as well 
```suggestion
            logger.debug("Error while decoding the resource string", e);
```

Reply:
Fixed with https://github.com/wso2/charon/pull/373/commits/409e73917431cc9b9552ead0e03d2f6525233db7